packages/components/src/components/dropdown/dropdown.jsx

```diff
@@ -418,11 +425,32 @@ const Dropdown = ({
                             onKeyPressed={onKeyPressed}
                             value={value}
                             parent_ref={dropdown_ref}
+                            suffix_icon={suffix_icon}
                         />
-                    )}
+
+                        {/* <MobileWrapper>
+                        <NativeSelect
+                            ref={native_select_ref}
+                            name={name}
+                            value={value}
+                            list={list}
+                            onChange={onChange}
+                        />
+                    </MobileWrapper> */}
+                    </div>
+                    {!error && hint && <p className='dc-dropdown__hint'>{hint}</p>}
                 </div>
-                {!error && hint && <p className='dc-dropdown__hint'>{hint}</p>}
-            </div>
+            </DesktopWrapper>
+            <MobileWrapper>
+                <SelectNative
```

Comment:
Just to confirm: this has the same functionality as `NativeSelect`? As I see that component accepted ref as well.

Reply:
yes as per zaki comment, we decided to change **NativeSelect** with **SelectNative**

Reply:
@kevinw-binary , i think u misunderstood my proposed solution. We should remove the `NativePicker` references inside the `Dropdown` since it's not being used anywhere anymore. 
In all cases currently we're using `SelectNative` and we don't need to use the `is_nativepicker` props anymore.

This separation between using `Dropdown` or `SelectNative` via `<DesktopWrapper>`, `<MobileWrapper>` should be handled outside of the `Dropdown` component, where the component is used.

Please refer to this example [here](https://github.com/binary-com/deriv-app/blob/master/packages/account/src/Sections/Profile/FinancialAssessment/financial-assessment.jsx#L366),